libs/core/src/lib/progress-indicator/progress-indicator.component.html

```diff
@@ -0,0 +1,58 @@
+<div
+    #progressIndicator
+    class="fd-progress-indicator"
+    tabindex="-1"
+    role="progressbar"
+    [attr.aria-valuemin]="valueMin"
+    [attr.aria-valuenow]="valueNow"
+    [attr.aria-valuemax]="valueMax"
+    [attr.aria-valuetext]="valueText"
+    [attr.aria-label]="valueText"
+    [ngClass]="state ? 'fd-progress-indicator--' + state : ''"
+>
+    <ng-container *ngIf="!hasPopover">
+        <ng-container *ngTemplateOutlet="indicatorContainer"></ng-container>
+    </ng-container>
+    <ng-container *ngIf="hasPopover">
+        <fd-popover [noArrow]="false" #popover>
+            <fd-popover-control>
+                <ng-container *ngTemplateOutlet="indicatorContainer"></ng-container>
+            </fd-popover-control>
+            <fd-popover-body>
+                <div class="fd-popover__wrapper">
+                    <div class="fd-progress-indicator__overflow">
+                        <span>{{ valueText }}</span>
+                        <span class="fd-progress-indicator__overflow-close" (click)="popover.close()">
+                            <i class="sap-icon sap-icon--decline"></i>
+                        </span>
+                    </div>
+                </div>
+            </fd-popover-body>
+        </fd-popover>
+    </ng-container>
+</div>
+
+<ng-template #indicatorContainer>
+    <div class="fd-progress-indicator__container">
+        <div
+            class="fd-progress-indicator__progress-bar"
+            [ngClass]="{ 'fd-progress-indicator__progress-bar--animated': animate }"
+            [ngStyle]="{ 'min-width': calculateProgressBarWidth() + '%', width: calculateProgressBarWidth() + '%' }"
+        >
+            <span class="fd-progress-indicator__label" *ngIf="calculateProgressBarWidth() > 50 && !valueText"
+                >{{ valueNow }}%</span
+            >
+            <span class="fd-progress-indicator__label" *ngIf="calculateProgressBarWidth() > 50 && valueText">{{
+                valueText
+            }}</span>
+        </div>
+        <div class="fd-progress-indicator__remaining">
+            <span class="fd-progress-indicator__label" *ngIf="calculateProgressBarWidth() <= 50 && !valueText"
+                >{{ valueNow }}%</span
+            >
+            <span class="fd-progress-indicator__label" *ngIf="calculateProgressBarWidth() <= 50 && valueText">{{
+                valueText
+            }}</span>
+        </div>
```

Comment:
```suggestion
    <div class="fd-progress-indicator__container" *ngIf="calculateProgressBarWidth() as progressBarWidth">
        <div
            class="fd-progress-indicator__progress-bar"
            [ngClass]="{ 'fd-progress-indicator__progress-bar--animated': animate }"
            [ngStyle]="{ 'min-width': progressBarWidth + '%', width: progressBarWidth + '%' }"
        >
            <span class="fd-progress-indicator__label" *ngIf="progressBarWidth > 50 && !valueText"
                >{{ valueNow }}%</span
            >
            <span class="fd-progress-indicator__label" *ngIf="progressBarWidth > 50 && valueText">{{
                valueText
            }}</span>
        </div>
        <div class="fd-progress-indicator__remaining">
            <span class="fd-progress-indicator__label" *ngIf="progressBarWidth <= 50 && !valueText"
                >{{ valueNow }}%</span
            >
            <span class="fd-progress-indicator__label" *ngIf="progressBarWidth <= 50 && valueText">{{
                valueText
            }}</span>
        </div>
```

Reply:
if `calculateProgressBarWidth() === 0` the template won't be rendered?